internal/auth/jwt.go

```
@@ -0,0 +1,90 @@
+	HeaderPrefix        = "X-"
+)
+
+type ErrorResponse struct {
+	Error   string `json:"error"`
+	Success bool   `json:"success"`
+}
+
+func jsonError(w http.ResponseWriter, message string, code int) {
+	w.Header().Set("Content-Type", "application/json")
+	w.WriteHeader(code)
+	json.NewEncoder(w).Encode(ErrorResponse{Error: message, Success: false})
```

golangci-lint (1.62.2)

Error return value of `(*encoding/json.Encoder).Encode` is not checked

(errcheck)